pkg/secrets-store/provider_client.go

```diff
@@ -67,9 +67,11 @@ const ServiceConfig = `
 
 var (
 	// PluginNameRe is the regular expression used to validate plugin names.
-	PluginNameRe        = regexp.MustCompile(`^[a-zA-Z0-9_-]{0,30}$`)
-	ErrInvalidProvider  = errors.New("invalid provider")
-	ErrProviderNotFound = errors.New("provider not found")
+	PluginNameRe = regexp.MustCompile(`^[a-zA-Z0-9_-]{0,30}$`)
+
+	ErrInvalidProvider       = errors.New("invalid provider")
+	ErrProviderNotFound      = errors.New("provider not found")
+	ErrMissingObjectVersions = errors.New("missing object versions")
```

Comment:
These error variables are only used in the `secretsstore` package. Can we unexport these?

Reply:
:+1: